powerline/matchers/vim/plugin/ctrlp.py

```diff
@@ -0,0 +1,54 @@
+# vim:fileencoding=utf-8
+from __future__ import (unicode_literals, division, absolute_import, print_function)
+
+try:
+    import vim
+except ImportError:
+    vim = object()
+
+import os
+
+from powerline.bindings.vim import buffer_name
+
+initialized = False
+def ctrlp(matcher_info):
+    name = buffer_name(matcher_info)
+    result = name and os.path.basename(name) == b'ControlP'
+    global initialized
+    if initialized:
+        return result
+    initialized = True
```

Comment:
Also it should not be launched from a matcher. After including two patches from my previous comments `initialize` function should be fine to be a `startup` attribute of ctrlp segments.